tango_simlib/utilities/fandango_json_parser.py

```diff
@@ -28,20 +28,10 @@
     'out_type_desc': 'doc_out'
 }
 
-class FandangoExportDeviceParser(object):
+class FandangoExportDeviceParser(Parser):
```

Comment:
- [ ] You  need to import class `Parser` from `tango_simlib.utilities.base_parser`.